tests/logic/test_pipeline_runs_logic.py

```diff
@@ -303,7 +304,12 @@ def test_get_result_and_download_pipeline_run_outputs(capture_logs):
     mock_job_report = mock({"status": "SUCCEEDED"})
     test_output_name = "output1"
     test_signed_url = "signed_url"
-    mock_pipeline_run_report = mock({"outputs": {test_output_name: test_signed_url}})
+    mock_pipeline_run_report = mock(
+        {
+            "outputs": {test_output_name: test_signed_url},
+            "output_expiration_date": str(datetime.now() + timedelta(days=1)),
```

Comment:
Is this the same format and time zone that the service returns?

Reply:
it is for both but it actually shouldnt matter.  thats why the logic converts w/e time it gets into local time zone and then does the comparison for exactly that reason.

Reply:
right, my concern was that if the string format was different, then the logic you have to convert the string time into a timestamp might not work